src/pickler/parser.clj

```diff
@@ -88,6 +93,14 @@
   [_ bb]
   {:type :tuple})
 
+(defmethod opcode 0x86
+  [_ bb]
+  {:type :tuple2})
```

Comment:
See https://github.com/python/cpython/blob/d42b61be88ea0b3eef6bab6602f5d836ea83d68a/Lib/pickle.py#L138
and https://github.com/python/cpython/blob/d42b61be88ea0b3eef6bab6602f5d836ea83d68a/Lib/pickle.py#L154

`:tuple` should be interpreted as "build tuple from topmost stack items" and `:tuple2` as "build 2-tuple from two topmost stack items". Concerning cyanite there's no difference at all since both opcodes are ignored anyway. It could as well be parsed as `:unsupported`.